client/lib/application/src/application-lifecycle/application-lifecycle.jsx

```diff
@@ -1,25 +1,124 @@
 import React from 'react';
-/*eslint-disable */
+import PropsExposer from 'common/src/components/pure/PropsExposer.jsx';
+import ComboBox from 'common/src/components/pure/ComboBox.jsx';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
```

Comment:
Nitpick: For consistency I'd rather have these in `common/src/` without the `components` folder in between (as there are already React components there this distinction makes no sense).